<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>X2Dome</class>
 <widget class="QDialog" name="X2Dome">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>736</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>736</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>736</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Lunatico Beaver Dome controller settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="X2Form" name="X2Form">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1024</width>
       <height>1024</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="SerialPortName" stdset="0">
      <string/>
     </property>
     <widget class="QGroupBox" name="DomeParams">
      <property name="geometry">
       <rect>
        <x>16</x>
        <y>97</y>
        <width>298</width>
        <height>176</height>
       </rect>
      </property>
      <property name="title">
       <string>Dome params</string>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>32</y>
         <width>160</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Home Position (Deg.) :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="homePosition">
       <property name="geometry">
        <rect>
         <x>184</x>
         <y>32</y>
         <width>88</width>
         <height>24</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="maximum">
        <double>359.990000000000009</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>32</x>
         <y>64</y>
         <width>144</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Park Postition (Deg.) :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="parkPosition">
       <property name="geometry">
        <rect>
         <x>184</x>
         <y>64</y>
         <width>88</width>
         <height>24</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="maximum">
        <double>359.990000000000009</double>
       </property>
      </widget>
      <widget class="QLabel" name="filePath">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>136</y>
         <width>264</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>/Users/foo/LunaticoBeaver_Rain.txt</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>104</y>
         <width>192</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Log rain sensor state</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="ControllerParams">
      <property name="geometry">
       <rect>
        <x>336</x>
        <y>8</y>
        <width>360</width>
        <height>448</height>
       </rect>
      </property>
      <property name="title">
       <string>Controllers settings</string>
      </property>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>24</y>
         <width>330</width>
         <height>200</height>
        </rect>
       </property>
       <property name="title">
        <string>Dome</string>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>8</x>
          <y>64</y>
          <width>200</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Number of steps per revolution :</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>16</x>
          <y>128</y>
          <width>192</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Rotation max speed (step/s) :</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>8</x>
          <y>160</y>
          <width>200</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Rotation acceleration (step/sÂ²) :</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QSpinBox" name="ticksPerRev">
        <property name="geometry">
         <rect>
          <x>216</x>
          <y>64</y>
          <width>104</width>
          <height>24</height>
         </rect>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="maximum">
         <number>999999999</number>
        </property>
       </widget>
       <widget class="QSpinBox" name="rotationSpeed">
        <property name="geometry">
         <rect>
          <x>216</x>
          <y>128</y>
          <width>104</width>
          <height>24</height>
         </rect>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
       </widget>
       <widget class="QSpinBox" name="rotationAcceletation">
        <property name="geometry">
         <rect>
          <x>216</x>
          <y>160</y>
          <width>104</width>
          <height>24</height>
         </rect>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="maximum">
         <number>999999999</number>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>24</x>
          <y>24</y>
          <width>281</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Calibrate</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="rotationMinSpeed">
        <property name="geometry">
         <rect>
          <x>216</x>
          <y>96</y>
          <width>104</width>
          <height>24</height>
         </rect>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>16</x>
          <y>96</y>
          <width>192</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Rotation min speed (step/s) :</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>232</y>
         <width>330</width>
         <height>200</height>
        </rect>
       </property>
       <property name="title">
        <string>Shutter</string>
       </property>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>16</x>
          <y>96</y>
          <width>192</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Shutter max speed (step/s) :</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>8</x>
          <y>128</y>
          <width>200</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Shutter acceleration (step/sÂ²) :</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="lowShutBatCutOff">
        <property name="geometry">
         <rect>
          <x>216</x>
          <y>160</y>
          <width>62</width>
          <height>22</height>
         </rect>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="value">
         <double>11.000000000000000</double>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>56</x>
          <y>160</y>
          <width>152</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Shutter voltage cutoff :</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QSpinBox" name="shutterSpeed">
        <property name="geometry">
         <rect>
          <x>216</x>
          <y>96</y>
          <width>104</width>
          <height>24</height>
         </rect>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
       </widget>
       <widget class="QSpinBox" name="shutterAcceleration">
        <property name="geometry">
         <rect>
          <x>216</x>
          <y>128</y>
          <width>104</width>
          <height>24</height>
         </rect>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="maximum">
         <number>999999999</number>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>24</x>
          <y>24</y>
          <width>281</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Calibrate</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_17">
        <property name="geometry">
         <rect>
          <x>16</x>
          <y>64</y>
          <width>192</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Shutter min speed (step/s) :</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QSpinBox" name="shutterMinSpeed">
        <property name="geometry">
         <rect>
          <x>216</x>
          <y>64</y>
          <width>104</width>
          <height>24</height>
         </rect>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QLabel" name="label_logo">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>8</y>
        <width>78</width>
        <height>80</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>Lunatico.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="X2_PhotoFileName" stdset="0">
       <string>Lunatico.png</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonCancel">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>416</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonOK">
      <property name="geometry">
       <rect>
        <x>184</x>
        <y>416</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QGroupBox" name="ControllerStatus">
      <property name="geometry">
       <rect>
        <x>16</x>
        <y>288</y>
        <width>298</width>
        <height>112</height>
       </rect>
      </property>
      <property name="title">
       <string>Status</string>
      </property>
      <widget class="QLabel" name="rainStatus">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>72</y>
         <width>96</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
      <widget class="QLabel" name="shutterBatteryLevel">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>48</y>
         <width>56</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>12.25 V</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>72</y>
         <width>112</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Rain sensor(s) :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>8</x>
         <y>48</y>
         <width>144</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Shutter voltage level :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="shutterPresent">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>24</y>
         <width>104</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>detected or not</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>8</x>
         <y>24</y>
         <width>144</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Shutter :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_logo_2">
      <property name="geometry">
       <rect>
        <x>144</x>
        <y>16</y>
        <width>128</width>
        <height>44</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>NexDome.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="X2_PhotoFileName" stdset="0">
       <string>NexDome.png</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>152</x>
        <y>72</y>
        <width>112</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Â© RTI-Zone 2021</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>X2Form</class>
   <extends>QFrame</extends>
   <header>x2form.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>8</number>
  </property>
  <property name="gridDeltaY">
   <number>8</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
